config/samples/custom-configs/mlpipeline-minio-artifact.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
may be related to https://github.com/opendatahub-io/data-science-pipelines-operator/pull/541/files#r1461979450

but i think we don't need this file anymore

Reply:
We still need it. There's an open issue to fix it: https://issues.redhat.com/browse/RHOAIENG-1720.